styles/main.scss

```diff
@@ -50,7 +56,6 @@ body {
 }
 
 .nypl-ds {
-  overflow: unset;
```

Comment:
Do we need this? I think we do need the overflow to be hidden(default) or it shows a horizontal bar on the landing page.

Reply:
I think it's ok for now.  This definitely will be a thing we're changing soon with the DS upgrade